project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -253,6 +253,11 @@ function c4m_features_og_group_dashboard_block_info() {
     'cache' => DRUPAL_CACHE_PER_PAGE,
   );
 
+  $blocks['c4m_follow_block'] = array(
+    'info' => t('C4M Follow group block'),
```

Comment:
@ordavidil this PR handles both cases, adding the Follow link to Projects and to Groups (see the changes on this file: https://github.com/capacity4dev/capacity4more/blob/cccb39c7a6363533768a63817d8e0ebbd8a3164c/project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.pages_default.inc )

I had issues to handle this functionality in both Projects and Groups ( https://issuetracker.amplexor.com/jira/browse/CFM-1358  and https://issuetracker.amplexor.com/jira/browse/CFM-1360 ) and I was told that might be better to tackle them together. Both issues are referencing this PR.

Should I split the PR in two PRs?